spec/integration/data_stream/fields/fields.spec.yml

```diff
@@ -318,6 +318,16 @@ spec:
         type: integer
         default: 1024
 
+      accept_ignored_values:
+        description: >
+          If set to true, elastic-package system tests will not fail if values
+          for this field can't be indexed for any incoming documents. This
+          should only be used if the failure is related to the test environment
+          and wouldn't happen in production, mitigate the issue via mapping or
+          pipeline changes otherwise. Defaults to false.
```

Comment:
Are there actual cases where we want to ignore these failures? Or this is a temporary measure till this issue is fixed in packages?

If there are actual cases where we want to ignore these failures, and as described here, this is only expected in tests, I think we should add this to the test config files. We have other test settings like `skip`, `numeric_keyword_fields` or `dynamic_fields` that are used to customize how validations behave on certain cases. I think this setting would fit there.

If this is a temporary measure to help merging https://github.com/elastic/elastic-package/pull/1738 before the issues are solved, we can think on other approaches. Something we have done in some cases is to apply new validations only on packages starting on a specific spec version. On these cases the issue is eventually fixed as packages upgrade their spec versions.

Reply:
> Are there actual cases where we want to ignore these failures? Or this is a temporary measure till this issue is fixed in packages?

There is at least one case in spring boot.

The test config files are indeed a better place, didn't think of those - I will adjust.